cypress/support/elements/common/Canvas.js

```diff
@@ -201,6 +205,16 @@ class Canvas {
       cy.wrap($handle).should('not.have.class', 'selected');
     });
   }
+
+  verifyTilesCopiedToDocument() {
+    // Wait for the document to be created and tiles to be copied
```

Comment:
I think this is a rather specfic function to be in the general `Canvas.js`, given that it's testing for the result of a particular copy operation. You could move it to the spec file since it's only likely to get used in this one test.
 
To test for a Text tile, you could make the code easier to read by using existing methods in `TextToolTile.js`, eg `getTextTile().should('have.length', 1)`